gitlab_sync.go

```diff
@@ -20,6 +20,7 @@ func shouldStartPipeline(branchName string) bool {
 		"staging",
 		"production",
 		"hosted",
+		"main",
```

Comment:
This will fix the issue that you are describing :+1: 

The rest of the changes are on features that I think we have disabled for Alvaldi. Nevertheless is no harm to add `main` for the cherry-pick suggestions and some tests.